sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -411,6 +406,7 @@ queryTerm
 
 queryPrimary
     : querySpecification                                                    #queryPrimaryDefault
+    | fromStatement                                                         #fromStmt
```

Comment:
According to Hive compatibility test, `fromStatement` should be in the same level as queries such as `SELECT * FROM table`.